src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionValue.cs

```diff
@@ -33,38 +33,38 @@ namespace Tizen.NUI.Scene3D
     /// <remarks>
```

Comment:
line 27 - "be used" -> "will be used" / "is used"
line 30 - "Each types" -> "Each type"

Reply:
Fix done.